libs/prelude/Language/Reflection.idr

```diff
@@ -396,9 +396,9 @@ mutual
   withNInj Refl = (Refl, Refl)
 
   private
-  instanceNInj : (InstanceN n xs = InstanceN n' ys) -> (n = n', xs = ys)
-  instanceNInj Refl = (Refl, Refl)
-
+  implementationNInj : (ImplementationN n xs = ImplementationN n' ys) -> (n = n', xs = ys)
```

Comment:
@zjhmale Ah I didn't notice that :). But please do it, if there were name changes in public functions. If it is possible to deprecate constructors as well, please do it for `InstanceN`/`ImplementationN` as well.